Libraries/LibGfx/Bitmap.h

```diff
@@ -114,18 +114,7 @@ class Bitmap : public RefCounted<Bitmap> {
 
     [[nodiscard]] size_t pitch() const { return m_pitch; }
 
-    [[nodiscard]] static unsigned bpp_for_format(BitmapFormat format)
-    {
-        switch (format) {
-        case BitmapFormat::BGRx8888:
-        case BitmapFormat::BGRA8888:
-            return 32;
-        default:
-            VERIFY_NOT_REACHED();
-        case BitmapFormat::Invalid:
-            return 0;
-        }
-    }
+    [[nodiscard]] static unsigned bpp_for_format(BitmapFormat);
```

Comment:
Looking at this again, I don't think `Bitmap::bpp()` nor `::bpp_for_format()` are being used anywhere? If that's the case, we could as well remove it. The element's size is already stored in `m_pitch`.

Reply:
At the time I started working on it, I'm sure it was used elsewhere. Let me check again

Reply:
Ok, done

Reply:
I'm keeping two commits just in case someone wants to revert this in the future

Reply:
Done